code/extensions/GoogleSitemapExtension.php

```diff
@@ -33,7 +33,8 @@ public function canIncludeInGoogleSitemap()
             $can = $this->owner->getGooglePriority();
         }
 
-        $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can);
+        // Allow override. In this case, since we don't know if this should be an "and" vs. "or" override, we'll use the LAST result (if any).
+        if ($override = $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can)) $can = array_pop($override);
```

Comment:
Min of what results? They're all booleans. And the concept (which I pointed out) was to take the last one to execute because there's no way to determine *which* should be used. How could we do what you're suggesting?